File: Python/programmers/1차비밀지도.py
def solution(n, arr1, arr2):
    answer = []
    for a, b in zip(arr1, arr2):
        t = bin(a | b)[2:]
        answer.append(
            "".join([" "] * (n - len(t)) + ["#" if i == "1" else " " for i in t])
        )
    return answer


# def solution(n, *maps):
#     return [line(n, a | b) for a, b in zip(*maps)]


# def line(n, x):
#     return ''.join(' #'[int(i)] for i in f'{x:016b}'[-n:])

File: C/programmers/1차비밀지도.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

vector<string> solution(int n, vector<int> arr1, vector<int> arr2 ) {
    vector<string> answer;
    
    for(int i = 0; i<n; i++){
        int a = arr1[i] | arr2[i];
        string ans;
        for(int j=0; j<n; j++){
            if (a & 1) ans = "#" + ans;
            else ans = ' ' + ans;
            a = a >>1;
        }
        answer.push_back(ans);
    }
    return answer;
}

int main(){
    // solution();
    
    std::cout << "test\n";
    return 0;
}